### PR TITLE
5.1.11 release documentation

### PR DESCRIPTION
**Pull request by mvalin**
### Basic info

- **JIRA issue**: https://sysdig.atlassian.net/browse/5_1_11_release_documentation

### Description

- Add Release History to link public document here
- Change the version and date
- Duplicate directory 5.1.10 as 5.1.11

### CODE REVIEW

File: 5.1/5.1.11/examples/rbac/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# RBAC for Installer (work in progress) v0.0.0a
```
Comment (radhikapc) [inline 5.1/5.1.11/examples/rbac/README.md:1]:
```suggestion
# RBAC for Installer v0.0.0a ((work in progress) 
```

**Review by radhikapc — CHANGES_REQUESTED** [review]
nits

**Review by radhikapc — APPROVED** [review]
LGTM

**Review by figarocorso — APPROVED** [review]
LGTM

**Pull request merged**